src/RedisProxy.php

```diff
@@ -261,10 +267,11 @@ public function get($key)
     /**
      * Set the string value of a key and return its old value
      * @param string $key
-     * @param string $value
-     * @return string|null null if key was not set before
+     * @param mixed  $value
```

Comment:
je to mixed? nie string?

Reply:
Ja som vsetky value pre istotu dal mixed, lebo jeden z tych 2 driverov pise ze podporuje nejaku unserializaciu ci co, takze to nemusi byt vzdy string? Ale je to na tebe, mozem dat string, nemam s tym problem, predpokladam ze to nic nepokasle? Ty asi lepsie vies ako sa tie funkcie pouzivaju.

Reply:
Dajme string, uvidíme O:-)